content/QRadar-SIEM/104.es.md

```diff
@@ -0,0 +1,175 @@
+
+    ```bash
+    ssh root@172.16.60.10
+    ```
+
+5.  En el indicador de contraseña, escriba "Q1d3m0".
+
+6.  Para generar eventos, escriba los siguientes comandos:
+
+    ```bash
+    cd /labfiles/extended/labfiles/logsources
+    ```
+
+    ```bash
+    /opt/qradar/bin/logrun.pl -f checkpoint1.syslog 1
+    ```
+
+    > El directorio /labfiles/extended/labfiles fue subido al servidor QRadar en los ejercicios **103: Configuración Avanzada de Laboratorio**.
+
+7.  Vuelva a la ventana del navegador.
+
+    > **Nota:** La Consola detectó tres eventos con la Categoría de Bajo Nivel ALMACENADO. Debido a que usamos el contexto localhost para estos eventos (127.0.0.1), el Nombre del Evento puede variar. En las capturas de pantalla de abajo se ve un Nombre de Evento de "Mensaje de Resultados de Búsqueda", pero también puede ser "Motor de Detección de Anomalías" u otros.
+
+8.  Para abrir cualquiera de los eventos, haga clic en el icono de **pausa** situado en la parte superior derecha de la Consola. Esta acción pausa la captura de eventos en tiempo real.
+
+9.  Para revisar los detalles del evento, haga doble clic en cualquiera de los tres eventos.
+
+    ![](images/104/image6.jpeg)
+
+10. En la página Detalles del evento, desplácese hasta la sección Información de la carga útil y haga clic en **Ajustar texto**.
+
+    ![](images/104/image7.jpeg)
+
+11. Analice la carga útil sin procesar y observe que las direcciones IP de origen y destino no se analizan correctamente comparando las direcciones con las que aparecen en la sección Información de origen y destino.
+
+En este punto, usted concluye que QRadar no procesó correctamente este evento. Puede determinar la fuente del registro generado y crear una fuente de registro adecuada. Además, en muchas ocasiones, si QRadar recibe suficientes eventos de registro para procesar, puede autodescubrir la fuente de registro y decidir la fuente de registro que mejor coincida con los eventos.
+
+## Visualización de fuentes de registro autodescubiertas
+
+QRadar autodescubre muchas fuentes de registro después de recibir varios registros de un tipo específico. No necesita configurar la fuente de registro para que QRadar reconozca los eventos de esa fuente de registro. En este ejercicio, analizará una fuente de registro autodescubierta.
+
+1.  Empiece por la Consola. Para cargar los filtros de búsqueda predeterminados, haga doble clic en la pestaña **Actividad de registro**.
+
+2.  A continuación, en el terminal SSH de QRadar, ejecute el siguiente script desde el directorio /labfiles/extended/labfiles/logsources:
+
+    ```bash
+    /opt/qradar/bin/logrun.pl -f checkpoint.syslog 10
+    ```
+
+3.  Vuelva a la ventana del navegador y observe los eventos que se muestran en la Consola.
+
+4.  Transcurridos unos 30 segundos, observará que la columna Event Name comienza a mostrar Firewall Permit, el LogSource se denomina Check Point y Low Level Category se actualiza a Firewall Permit.
+
+    ![](images/104/image8.jpeg)
```

Comment:
when not using `./images...` I am not seeing images on the public VEST site, just when in development. I am doing a PR now for the English version that uses an explicit relative path.

Reply:
this appears to have been a cache issue on the workstation where I was testing the site content. marking resolved.